src/components/shared/Project/ProjectHeader/index.tsx

```diff
@@ -25,6 +25,8 @@ export default function ProjectHeader({
   const headerHeight = 120
   const spacing = 20
 
+  const projectTitle = metadata?.name || t`Untitled project`
```

Comment:
super tiny nit: for nullish checks does it make more sense to use `??`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator